de1plus/machine.tcl

```diff
@@ -505,7 +506,18 @@ array set ::de1_substate_types {
 	4	"preinfusion"
 	5	"pouring"
 	6	"ending"
+	7	"Steaming"
```

Comment:
I think CamelCase vs all-lowercase got mixed up here.

Reply:
What would you prefer? 

The names I inserted are right out of the struct. If the API is the struct, then those are correct. See `APIDataTypes.hpp`

Regrettably, changing the others at this point is a major refactor, as they have been used extensively as keys throughout the current app.

```
  Sleep,      // 0 Everything is off
  GoingToSleep, // 1
  Idle,       // 2 Heaters are controlled, tank water will be heated if required.
  Busy,       // 3 Firmware is doing something you can't interrupt (eg. cooling down water heater after a shot, calibrating sensors on startup).
  Espresso,   // 4 Making espresso
  Steam,      // 5 Making steam
  HotWater,   // 6 Making hot water
  ShortCal,   // 7 Running a short calibration
  SelfTest,   // 8 Checking as much as possible within the firmware. Probably only used during manufacture or repair.
  LongCal,    // 9 Long and involved calibration, possibly involving user interaction. (See substates below, for cases like that).
  Descale,    // A Descale the whole bang-tooty
  FatalError, // B Something has gone horribly wrong
  Init,       // C Machine has not been run yet
  NoRequest,  // D State for T_RequestedState. Means nothing is specifically requested
  SkipToNext, // E In Espresso, skip to next frame. Others, go to Idle if possible
  HotWaterRinse, // F Produce hot water at whatever temperature is available
  SteamRinse,    // 10 Produce a blast of steam
  Refill,        // 11 Attempting, or needs, a refill.
  Clean,         // 12 Clean group head
  InBootLoader,  // 13 The main firmware has not run for some reason. Bootloader is active.
  AirPurge,      // 14 Air purge.
  SchedIdle      // 15 Scheduled wake up idle state
```